packages/indexer-common/src/indexer-management/resolvers/cost-models.ts

```diff
@@ -121,22 +146,17 @@ export default {
     } catch (err) {
       throw new Error(`Invalid cost model or variables: ${err.message}`)
     }
-    const [model] = await models.CostModel.findOrBuild({
+    const [oldModel] = await models.CostModel.findOrBuild({
```

Comment:
Probably better not to build. Just try to find the latest model, and if there isn't one don't try to copy values over.